Application/EdFi.Ods.Api/Security/AuthorizationStrategies/AuthorizationStrategyNameAttribute.cs

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using EdFi.Ods.Common.Security.Authorization;
+
+namespace EdFi.Ods.Api.Security.AuthorizationStrategies;
+
+/// <summary>
+/// Provides the authorization strategy name that is associated with an <see cref="IAuthorizationStrategy" /> implementation
+/// (replacing the previously used class naming convention). 
+/// </summary>
+[AttributeUsage(AttributeTargets.Class, Inherited = false)]
+public sealed class AuthorizationStrategyNameAttribute : Attribute
```

Comment:
We are no longer using a naming-convention based mapping from authorization strategy name (from security metadata) to the implementation class. This must now be specified explicitly through the use of this attribute. This change allows for the `CustomViewBasedAuthorizationStrategy` to exist with different behavioral characteristics from the original strategy implementations.